pkgs/applications/networking/browsers/w3m/cygwin.patch

```diff
@@ -0,0 +1,1710 @@
+diff -ur w3m-0.5.3/config.guess new/w3m-0.5.3/config.guess
```

Comment:
Was this just generated by autoconf? Might be simpler (and easier to review) to just call autoconf as part of the build, if that doesn't break stuff